cmd/aries-agentd/startcmd/start.go

```diff
@@ -158,12 +186,14 @@ func startAgent(server server, host, inboundHost, dbPath string) error {
 		router.HandleFunc(handler.Path(), handler.Handle()).Methods(handler.Method())
 	}
 
-	logger.Infof("Starting aries agentd on host [%s]", host)
+	logger.Infof("Starting aries agentd on host [%s]", parameters.host)
+
+	webhook.StartWebhookDispatcher(parameters.webhookURLs)
```

Comment:
move this inside restapi.New()

If no webhook urls provided, print warning and don't start dispatcher.

Ask George/Troy if it should be a mandatory argument. @llorllale @troyronda

Reply:
First two sentences: done.